midi_app_controller/gui/utils.py

```diff
@@ -74,6 +74,11 @@ def showPopup(self):
         self.refresh_items()
         super().showPopup()
 
+    def refresh(self):
+        """Refreshes the list of items."""
+        self.clear()
+        self.addItems(self.get_items())
```

Comment:
what is this for? There is already `refresh_items` which makes sure not to deselect the item already selected by user, I think it does everything you'd want, so maybe rename `refresh_items` to `refresh` if it's a clearer name?

Reply:
Oops now that I look at it, it's probably a mistake from merging. Anyway, please remove